subprojects/robotpy-wpiutil/gen/RawFrame.yml

```
@@ -0,0 +1,20 @@
+---
+
+enums:
+  WPI_PixelFormat:
+    ignore: true
+  WPI_TimestampSource:
+    value_prefix: WPI_TIMESRC
```

I think this currently spits out the members prefixed with underscores.
```suggestion
    value_prefix: WPI_TIMESRC_
```

---

Based on my reading of https://github.com/robotpy/robotpy-build/blob/main/robotpy_build/autowrap/cxxparser.py#L538 , it will add an _ for me?

---

Oh, huh, I didn't recall that being a thing. Fair enough!